src/main/resources/assets/js/app/view/context/ContextView.ts

```diff
@@ -258,7 +259,7 @@ export class DetailsView extends api.dom.DivEl {
 
     private initCommonWidgetViews() {
 
-        this.defaultWidgetView = WidgetView.create().setName(i18n('field.details')).setDetailsView(this)
+        this.defaultWidgetView = WidgetView.create().setName(i18n('field.contextPanel')).setContextView(this)
```

Comment:
Not a very correct name for this resource - should be similar to the two other widgets. Let's rename this to ``field.contextPanel.details`` and let's change the text itself from "_Properties_" to "_Details_".